image/lib.sh

```diff
@@ -25,23 +32,68 @@ function errnullify     { "$@" 2>/dev/null; }
 : ${RUNTIME_DEFERRED:="$IMAGE_ROOT/runtime.d"}
 
 ##
-## Utils
+## Utils (must be in-process, hence placement here).
```

Comment:
Added a few utils to lib, as they're generic, useful, and I'd rather stop duplicating them in scripts.